src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -1446,29 +1451,20 @@
                         <Trigger Property="IsEnabled" Value="true">
                             <Setter TargetName="text" Property="Foreground" Value="{StaticResource Blue300Brush}" />
                         </Trigger>
-
-                        <DataTrigger Binding="{Binding ElementName=text, Path=Text}" Value="+">
-                            <Setter TargetName="text" Property="FontSize" Value="28px" />
-                            <Setter TargetName="text" Property="Margin" Value="1,3,0,0" />
-                            <Setter TargetName="text" Property="Foreground" Value="#999999" />
-                            <Setter TargetName="container" Property="Height" Value="22px" />
-                            <Setter TargetName="container" Property="Width" Value="22px" />
-                            <Setter TargetName="roundedBorder" Property="CornerRadius" Value="4,0,0,4" />
-                        </DataTrigger>
-                        <DataTrigger Binding="{Binding ElementName=text, Path=Text}" Value="-">
-                            <Setter TargetName="text" Property="FontSize" Value="28px" />
-                            <Setter TargetName="text" Property="Margin" Value="0,3,1,0" />
-                            <Setter TargetName="text" Property="Foreground" Value="#999999" />
-                            <Setter TargetName="container" Property="Height" Value="22px" />
-                            <Setter TargetName="container" Property="Width" Value="22px" />
-                            <Setter TargetName="roundedBorder" Property="CornerRadius" Value="0,4,4,0" />
-                        </DataTrigger>
                     </ControlTemplate.Triggers>
                 </ControlTemplate>
             </Setter.Value>
         </Setter>
     </Style>
 
+    <Style TargetType="{x:Type nodes:DynamoNodeButton}" BasedOn="{StaticResource SNodeTextButton}" />
```

Comment:
A default style for DynamoNodeButton, an alternative to the style set in the constructor, to allow the style to be overridden.